backends/tfhe-cuda-backend/cuda/src/pbs/bootstrap_multibit.cu

```diff
@@ -354,6 +307,84 @@ void scratch_cuda_multi_bit_pbs_64(
   }
 }
 
+void scratch_split_cuda_multi_bit_pbs_64(
```

Comment:
Naming is always `cuda_` or `scratch_cuda_`, so here it should be `scratch_cuda_split_multi_bit_pbs_64` I think, to be consistent.

Reply:
I forgot about these `_split_` in the code. I will remove it.

Reply:
However, here we need a different name for the generic algorithm. Otherwise this function will conflict with the main entry point.
I thing `_ generic_` fits better. I will update everything to follow the naming convention.

Reply:
Wait, I'm not sure what you mean.

Reply:
Wouldn't `_split_` be better, to match classical PBS naming?